packages/core/src/browser/secondary-window-handler.ts

```diff
@@ -195,18 +194,18 @@ export class SecondaryWindowHandler {
         return undefined;
     }
 
-    protected addWidget(widget: ExtractableWidget): void {
+    protected addWidget(widget: ExtractableWidget, win: Window): void {
```

Comment:
`window` is the name of the global window object. That means the compiler will not complain when I accidentally refer to the global object instead of the function argument. So I try to avoid that name (same for `document`, etc.)